openadmet_toolkit/structure/cofolding.py

```diff
@@ -12,13 +12,12 @@
 
 
 def combine_seq_smiles_to_fasta(
-    seq_string: str,
-    smiles: str,
-    protein_name: str = "protein",
-    ligand_name: str = "ligand",
+    seqs: list,
```

Comment:
Use typing of the lists here and below 

e.g 

```suggestion
    seqs: list[str]
```